examples/async-examples.js

```diff
@@ -0,0 +1,59 @@
+(async () => {	
+	const revai = require('../../revai-node-sdk');
```

Comment:
indent is too large. Is this a tab or space? It looks like a double tab

Reply:
Convention is to scope variables to their limited use; however, with imports (and import variables) I usually see them being scoped to the file rather than being imported inline dynamically. Scoping to the file level would allow you use the import ('revai') in private helper functions defined outside the scope of the main anon async function. 


Additionally I think we should structure the examples like this
```
revai-node-sdk/
├── src/
├── test/
└── examples/
    ├── async_transcribe_local_media_file/
    │   ├── async_transcribe_local_media_file.js
    │   └── package.json
    ├── async_transcribe_media_from_url/
    │   ├── async_from_local_file.js
    │   └── package.json
    ├── streaming_transcribe_from_local_file/
    │   ├── streaming_transcribe_from_local_file.js
    │   └── package.json
    ├── streaming_transcribe_from_stream/
    │   ├── streaming_transcribe_from_stream.js
    │   └── package.json
    └── streaming_transcribe_from_live_microphone/
        ├── streaming_transcribe_from_live_microphone.js
        └── package.json
```


Then we can use the imports without using the relative path

```suggestion
	const revai = require('revai-node-sdk');
```


My reason for

Reply:
For Kyle: I asked Albert to stop working on the microphone nodejs streaming example because nodejs doesn't have a nice microphone package. I think it would be more valuable to create a client/server app that is basically simplified homepage streaming demo.

Reply:
Another comment; can we update the readme.md to let customers know that we have examples.

Reply:
Also, we should write down which version of node we support. Or is this something we can add as a travis ci type badge to the readme?

Reply:
Would a client server app with a streaming demo not require getting audio from microphone anyways? Can you clarify the specs of the app example? 

It should say in the readme, please add it if it doesn't. We support node 8 and 10.

Reply:
I believe by using a client server app would allow microphone access to come via the browser rather than directly from node.